src/test/regress/expected/propagate_extension_commands.out

```diff
@@ -443,5 +493,133 @@ SELECT run_command_on_workers($$SELECT count(*) FROM pg_extension WHERE extname
  (localhost,57638,t,2)
 (2 rows)
 
+-- Check the pg_dist_object on the both nodes
+SELECT run_command_on_workers($$
+SELECT count(*)
+FROM citus.pg_dist_object, pg_proc
+WHERE pg_proc.proname = 'seg_in' and pg_proc.oid = citus.pg_dist_object.objid;
+$$);
+ run_command_on_workers
+---------------------------------------------------------------------
+ (localhost,57637,t,1)
+ (localhost,57638,t,1)
+(2 rows)
+
+DROP EXTENSION seg CASCADE;
+-- Recheck the pg_dist_object
+SELECT pg_proc.proname as DistributedFunction
+FROM citus.pg_dist_object, pg_proc
+WHERE pg_proc.proname = 'seg_in' and pg_proc.oid = citus.pg_dist_object.objid;
```

Comment:
oid is only unique per classid, so these joins could start returning random pg_dist_object records that happen have the same OID. We should add a classid = 'pg_proc'::regclass filter.

Reply:
Yes, I did check for pg_extension but missed for pg_proc, Thanks